src/include/rbd_types.h

```diff
@@ -68,6 +68,23 @@
 #define RBD_MAX_OBJ_NAME_SIZE	96
 #define RBD_MAX_BLOCK_NAME_SIZE 24
 
+/**
+ * Maximum string length of the RBD v2 image id (not including
+ * null termination). This limit was derived from the existing
+ * RBD_MAX_BLOCK_NAME_SIZE limit which needs to hold the "rbd_data."
+ * prefix and null termination.
+ */
+#define RBD_MAX_IMAGE_ID_LENGTH 14
+
+/**
+ * Maximum string length of the RBD block object name prefix (not including
+ * null termination).
+ *
+ * v1 format: rb.<max 8-byte high id>.<max 8-byte low id>.<max 8-byte extra>
+ * v2 format: rbd_data.[<max 19-byte pool id>.]<max 14-byte image id>
+ */
+#define RBD_MAX_BLOCK_NAME_PREFIX_LENGTH 43
```

Comment:
Perhaps RBD_MAX_OBJECT_PREFIX_LENGTH, so that it matches the method name?  I'd also mention that it might change in the future when new features are added.